app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -387,9 +387,12 @@ You can select by class:
 ```html
 <dom-module id="x-foo">
   <template>
-    <style>
-      ::slotted(.green) {
-        color: green;
+    <style> 
+      <!-- To work within the Shady CSS scoping shim limitations,
+           and to ensure consistent cross-browser behavior,
+	   add a selector to the left of the ::slotted(.classname) notation. --> 
```

Comment:
Can you have a look at this one: http://plnkr.co/edit/suVhRDAB5Rq7Q7T6zbMI?p=preview